doc/extdev/event_callbacks.rst

```diff
@@ -52,21 +52,25 @@ Below is an overview of the core event that happens during a build.
 
    10. event.env-updated(app, env)
    11. event.env-get-updated(app, env)
-   12. event.env-check-consistency(app, env)
+
+   if environment is written to disk:
+      12. event.env-check-consistency(app, env)
+
+   13. event.write-starting(app, builder)
```

Comment:
Maybe add a comment saying that this is done after determining whether to write in parallel or serial mode (i.e., `app.parallel_ok` is set correctly and should not be changed at that time).

Reply:
done